spec/lib/tmuxinator/window_spec.rb

```diff
@@ -208,6 +210,69 @@
     end
   end
 
+  describe "#synchronize" do
+    context "synchronization enabled" do
+      let(:synchronize) { true }
+
+      it "should have synchronization enabled" do
+        expect(window.synchronize).to be true
+      end
+    end
+
+    context "synchronization disabled" do
+      let(:synchronize) { false }
+
+      it "should have synchronization enabled" do
+        expect(window.synchronize).to be false
+      end
+    end
+
+    context "synchronization not specified" do
+      it "should have synchronization disabled" do
+        expect(window.synchronize).to be false
+      end
+    end
+  end
+
+  describe "#tmux_synchronize_panes" do
+    let(:project) { double(:project) }
+    let(:window) { Tmuxinator::Window.new(yaml, 0, project) }
+    let(:root?) { true }
+    let(:root) { "/project/tmuxinator" }
+
+    before do
+      allow(project).to receive_messages(
+        name: "test",
+        tmux: "tmux",
+        root: root,
+        root?: root?,
+        base_index: 1
+      )
+    end
+
+    let(:tmux_part) { project.tmux }
+    let(:window_option_set) { "set-window-option" }
+    let(:target_part) { "-t #{window.tmux_window_target}" }
+    let(:synchronize_panes) { "synchronize-panes" }
+    let(:tmux_part) { project.tmux }
+
+    context "synchronization enabled" do
+      let(:synchronize) { true }
+      let(:enabled) { "on" }
+
+      let(:full_command) do
+        command = <<END_COMMAND
+#{tmux_part} #{window_option_set} #{target_part} #{synchronize_panes} #{enabled}
```

Comment:
And this is because the code linter doesn't like more than 80 characters. I'm not a ruby expert. Any suggestions for a better way to implement that line?